specification/redhatopenshift/resource-manager/readme.python.md

```diff
@@ -15,12 +15,12 @@ no-namespace-folders: true
 
 Generate all API versions currently shipped for this package
 
-```yaml $(python)
-multiapi: true
+```yaml $(python) && $(multiapi)
```

Comment:
@ventifus Please revert change for this line otherwise python pipeline can't work

Reply:
Ok thank you.